chapter_recurrent-modern/seq2seq.md

```diff
@@ -139,13 +211,33 @@ X = np.array([[1, 2, 3], [4, 5, 6]])
 npx.sequence_mask(X, np.array([1, 2]), True, axis=1)
 ```
 
+```{.python .input}
+#@tab pytorch
+#@save
+def sequence_mask(X, valid_len, value=0):
+    output = X.clone()
+    for count, matrix in enumerate(output):
+        matrix[int(valid_len[count]):]=value
+    return output
+
+X = torch.tensor([[1, 2, 3], [4, 5, 6]])
+sequence_mask(X, torch.tensor([1, 2]))
+```
+
 Apply to $n$-dim tensor $X$, it sets `X[i, len[i]:, :, ..., :] = 0`. In addition, we can specify the filling value such as $-1$ as shown below.
 
 ```{.python .input  n=8}
 X = np.ones((2, 3, 4))
 npx.sequence_mask(X, np.array([1, 2]), True, value=-1, axis=1)
 ```
 
+```{.python .input}
+#@tab pytorch
+X = torch.ones(2, 3, 4)
```

Comment:
```suggestion
X = d2l.ones(2, 3, 4)
```
We are now making a few functions standard across all frameworks. See [here](https://github.com/d2l-ai/d2l-en/blob/28b0d4ce61c3f6f79e95f57d0be284c71453f9f4/d2l/torch.py#L35)